### PR TITLE
Test OffscreenCanvas's transferToImageBitmap function for both webgl and webgl2

### PR DESCRIPTION
**Pull request by xlai-o**
These two tests are based on an existing Chromium Layout test for OffscreenCanvas webgl (https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/fast/canvas/webgl/offscreenCanvas-transferToImageBitmap-texImage2D.html). One tests for webgl 1 and the other tests for webgl 2. They share the same functions provided in sdk/tests/js/tests/offscreencanvas-transfer-image-bitmap.js.

@zhenyao for review.

### CODE REVIEW

File: sdk/tests/js/tests/offscreencanvas-transfer-image-bitmap.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+}
+
+function _checkCanvas(buf, width, height, webglContextVersion)
+{
+    for (var i = 0; i < width * height; i++) {
+        if (buf[i * 4] != 255 || buf[i * 4 + 1] != 255 ||
+            buf[i * 4 + 2] != 0 || buf[i * 4 + 3] != 255) {
+            testFailed("OffscreenCanvas." + webglContextVersion +
+                ": This pixel should be red, but it is: [" + buf[i * 4] + ", " +
```
Comment (kainino0x) [inline sdk/tests/js/tests/offscreencanvas-transfer-image-bitmap.js]: This looks like magenta, not red. Maybe say "this pixel should be [255, 255, 0, 255]" to be exact?

Comment (xlai-o) [inline sdk/tests/js/tests/offscreencanvas-transfer-image-bitmap.js]: Thanks I've changed this part.

File: sdk/tests/conformance/offscreencanvas/00_test_list.txt [hunk 2]
```diff
@@ -8,3 +8,4 @@ context-lost-restored-worker.html
 methods.html
 methods-worker.html
 offscreencanvas-resize.html
+offscreencanvas-transfer-image-bitmap.html
```
Comment (kenrussell) [inline sdk/tests/conformance/offscreencanvas/00_test_list.txt]: Could you please add --min-version 1.0.4?

File: sdk/tests/conformance2/offscreencanvas/00_test_list.txt [hunk 3]
```diff
@@ -2,3 +2,4 @@ context-creation.html
 context-creation-worker.html
 methods-2.html
 methods-2-worker.html
+offscreencanvas-transfer-image-bitmap.html
```
Comment (kenrussell) [inline sdk/tests/conformance2/offscreencanvas/00_test_list.txt]: Could you please add --min-version 2.0.1?

**Comment by kenrussell** [comment]
Looks good Olivia. Could you add the conformance test versions? Let's merge afterward. Thanks.

File: sdk/tests/conformance/offscreencanvas/00_test_list.txt (hunk 2 above)
Comment (xlai-o) [inline sdk/tests/conformance/offscreencanvas/00_test_list.txt]: Thanks @kenrussell  I've modified this part.

**Comment by kenrussell** [comment]
Very nice. Thanks @xlai-o for the solid tests. LGTM.

**Pull request merged**